extension/settings/settings.css

```diff
@@ -17,10 +17,6 @@ input[type='file'] {
   display: none;
 }
 
-input[type='number'] {
```

Comment:
Might be worth looking into the future to see how much of this css file we can put into Settings.tsx for simplification.